src/LoginForm.jsx

```diff
@@ -0,0 +1,63 @@
+import { sendSignInLinkToEmail } from 'firebase/auth';
+import { auth } from './utils/firebaseConfig';
+import Header from './components/Header';
+import { ToastContainer, toast } from 'react-toastify';
+
+const LoginForm = ({ userEmail, setUserEmail, isLoading, setIsLoading }) => {
+  const notifyError = (message) => toast.error(message);
+  const notifyInfo = (message) => toast.success(message);
+
+  const handleLogin = async (event) => {
+    event.preventDefault();
+    setIsLoading(true);
+    try {
+      await sendSignInLinkToEmail(auth, userEmail, {
+        url: 'https://trivia-quiz-develop.vercel.app/',
+        handleCodeInApp: true,
+      });
+      localStorage.setItem('email', userEmail);
+      notifyInfo('We have sent you an email with a link to sign in');
```

Comment:
another hardcoded string - should go to constants at this stage. Later it could be improved further with i18n.